studio/stores/authConfig/schema/AuthProviders/AuthProvidersFormValidation.tsx

```diff
@@ -243,6 +243,21 @@ const PROVIDER_PHONE = {
         matches: ['twilio', 'messagebird', 'textlocal', 'vonage'],
       },
     },
+    SMS_TEST_OTP: {
+      type: 'string',
+      title: 'Test Phone Numbers and OTPs',
+      description:
+        'Register phone number and OTP combinations for testing as a comma separated list of <phone number>=<otp> pairs. Example: `18005550123=789012`',
+    },
+    SMS_TEST_OTP_VALID_UNTIL: {
+      type: 'datetime',
+      title: 'Test OTPs Valid Until',
+      description:
+        "Test phone number and OTP combinations won't be active past this date and time.",
```

Comment:
can we also include the timezone that we're showing in the input field/date picker?

`Test phone number and OTP combinations won't be active past this date and time. The date and time shown in the field is formatted to your local time zone (+XXXX)`

can use dayjs to generate the TZ - something like `dayjs().format('ZZ')`